src/main/java/microsoft/exchange/webservices/data/core/request/SingletonEwsExecutor.java

```diff
@@ -0,0 +1,48 @@
+package microsoft.exchange.webservices.data.core.request;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import javax.annotation.PreDestroy;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+public class SingletonEwsExecutor {
```

Comment:
Should probably create some tests around this

Reply:
Added a test to test singleton